src/containers/auth/login.js

```diff
@@ -30,3 +36,9 @@ function getCookies(cookieString) {
             {}
         )
 }
+
+function getExtensionInfo() {
+    return new Promise(resolve =>
+        sendMessage(null, { action: 'getExtensionInfo' }, resolve)
```

Comment:
If `extensionId` above is no longer needed, it seems like this could be removed.